test/orbit_web/controllers/sign_in_controller_test.exs

```diff
@@ -105,6 +105,36 @@ defmodule OrbitWeb.SignInControllerTest do
              } = json_response(conn, 200)
     end
 
+    @tag :authenticated
+    test "shows official's email address if employee names are empty string", %{conn: conn} do
```

Comment:
There's unit tests for `display_name` in `test/orbit/employee_test.exs`. I think the test for this logic would be better there, since how the name is generated isn't important to to the controller.

Reply:
Agree 969537e